dev/com.ibm.ws.microprofile.config.1.1/test/src/com/ibm/ws/microprofile/config/extended/test/WebSphereConfigTest.java

```diff
@@ -48,39 +40,12 @@ public void testOptionalString() throws NoSuchFieldException, SecurityException
 
         WebSphereConfig config = (WebSphereConfig) builder.build();
 
-        Type type = TestTypes.class.getField("optString").getAnnotatedType().getType();
-
-        Object value = config.getValue("key1", type, true);
-        assertTrue(value instanceof Optional);
-        Optional<String> optValue = (Optional<String>) value;
-        assertTrue(optValue.isPresent());
-        assertEquals("value1", optValue.get());
+        Object value = config.getValue("key5", String.class, true);
+        assertNull("Value should have been null: " + value, value);
     }
 
-    @SuppressWarnings("unchecked")
     @Test
-    public void testOptionalURL() throws NoSuchFieldException, SecurityException {
```

Comment:
What was this test originally for, and why is it being removed?

Reply:
This was testing that an Optional<URL> could be created using the internal WebSphereConfig interface ... I changed that interface so it isn't possible any more. If an instance of Optional is needed then the standard getOptionalValue method should be used.